tests/vmi_configuration_test.go

```diff
@@ -1661,8 +1661,7 @@ var _ = Describe("[sig-compute]Configurations", func() {
 			return updatedCPUName
 		}
 
-		// Collect capabilities once for all tests
-		tests.BeforeAll(func() {
+		BeforeEach(func() {
```

Comment:
Can the two `beforeEach` (line 1664 and line 1669) be merged

Reply:
Done.